scripts/model-discovery/check-new-models.py

```
@@ -266,11 +266,17 @@ def fetch_together_models() -> list[str]:
     if not api_key:
         return []
     try:
-        response = requests.get('https://api.together.ai/models', headers={'Authorization': f'Bearer {api_key}'})
+        response = requests.get('https://api.together.ai/v1/models', headers={'Authorization': f'Bearer {api_key}'})
```

Ruff (0.15.17)

[error] Probable use of `requests` call without timeout

(S113)